tests/set-namespace/replace-match/Kptfile

```diff
@@ -0,0 +1,10 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: example
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/set-namespace:unstable
+      configMap:
+        namespace: new-ns
+        namespaceMatcher: example2
```

Comment:
Why this is a test instead of an example?

Reply:
yeah, example fits better here. I'd prefer updating the tests/examples and docs in a separate PR. It's more efficient to think them together.

Reply:
> I'd prefer updating the tests/examples and docs in a separate PR.

This seems to be a completely **new** test. I don't see the benefits of adding it as test first and then changing it to an example. For new example or test, I'd suggest to create it in the right place in the beginning.